modules/rotating-certificate-authority.adoc

```diff
@@ -6,42 +6,65 @@
 [id="rotating-certificate-authority_{context}"]
 = Rotating the etcd certificate
 
-Rotate the `etcd` certificate before it expires.
+Manually rotate the `etcd` certificate before it expires.
 
 .Procedure
 
-. Verify the remaining lifetime of the new signer certificate by running the following command:
+. Make a backup copy of the current signer certificate by running the following command:
 +
 [source,terminal]
 ----
-$ oc get secret -n openshift-etcd etcd-signer -ojsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
+$ oc get secret -n openshift-etcd etcd-signer -oyaml > signer_backup_secret.yaml
```

Comment:
don't forget to backport this to 4.17 as well! I think this might save us a few support tickets :)